dictionary/dictionary.py

```diff
@@ -0,0 +1,67 @@
+import mysql.connector
+from difflib import get_close_matches
+
+# connects to database
+con = mysql.connector.connect(
+    user="ardit700_student",
+    password="ardit700_student",
+    host="108.167.140.122",
+    database="ardit700_pm1database"
+)
```

Comment:
Take these database connection parameters, download MySQL Workbench 8.0 for Windows. Set up the connection to the database using the parameters from here.
Watch the first hour of this video. Try to apply the MySQL commands from the video to the Dictionary database.
https://www.youtube.com/watch?v=7S_tz1z_5bA